Source/Extensions/Blazorise.Video/wwwroot/video.js

```diff
@@ -478,11 +478,11 @@ function registerToEvents(dotNetAdapter, player) {
     });
 
     player.addEventListener('seeking', (event) => {
-        invokeDotNetMethodAsync(dotNetAdapter, "NotifySeeking");
+        invokeDotNetMethodAsync(dotNetAdapter, "NotifySeeking", event.detail || 0);
```

Comment:
`event.detail || 0`
So in js if we use double pipe it coalesces the value? so if event.detail is null, 0 will be sent?

Reply:
Yeah. I don't think null is valid value in vidstack but this is just a precaution. 0 can be associated with zero time in the player.